scripts/export-import-test.py

```diff
@@ -67,30 +95,37 @@ def main():
 
     script_dir = os.path.dirname(os.path.realpath(__file__))
     kuzu_root = os.path.abspath(os.path.join(script_dir, ".."))
-    current_branch = run_command(
-        "git rev-parse --abbrev-ref HEAD", cwd=kuzu_root, capture_output=True
-    )
+    base_worktree = os.path.join(kuzu_root, ".worktree-base")
+    test_worktree = os.path.join(kuzu_root, ".worktree-test")
+    create_worktree(base_worktree, base_commit, kuzu_root)
+    create_worktree(test_worktree, test_commit, kuzu_root)
     try:
-        run_command(f"git checkout {base_commit}", cwd=kuzu_root)
-
-        version = run_command(
-            f"python3 benchmark/version.py", cwd=kuzu_root, capture_output=True
-        )
+        version = get_version(base_worktree)
         if version == "0":
             raise Exception("Failed to determine version. Aborting.")
-        export_path = os.path.join(output_dir, version)
+        export_path = os.path.abspath(os.path.join(output_dir, version))
```

Comment:
minor issue: `export_path` is being used in `finally` but may not have been initialized if the above exception occurs.